pelican/readers.py

```diff
@@ -136,13 +136,19 @@ def __init__(self, *args, **kwargs):
 
     def _parse_metadata(self, document):
         """Return the dict containing document metadata"""
+        if 'FORMATTED_FIELDS' in self.settings:
```

Comment:
So if you put this setting in settings.py as well, you can just rely on it being there. Additionally, I wouldn't hardcode 'summary' being always parsed. Perhaps make it a default?